slangpy/reflection/reflectiontypes.py

```
@@ -1761,6 +1763,24 @@ def can_convert_to_int(value: Any):
         return False
 
 
+def can_convert_to_bool(value: Any):
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)